docs/en/latest/plugins/limit-conn.md

```diff
@@ -141,3 +141,79 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
     }
 }'
 ```
+
+## Limit the rate of WebSocket connections
```

Comment:
I think you can simplify this step. We don't have to tell the user how to create a WebSocket Server.